### PR TITLE
Re-add support for STM32F411xC/xE option bytes read/write

### PR DESCRIPTION
**Pull request by AkshayPatwardhan**
Fix STM32F411xC/xE chip file.

For some reason, support for STM32F411 was removed along the way? As far as I can tell - this was supported via #1141  but looks like it got removed along the way.

### CODE REVIEW

**Comment by AkshayPatwardhan** [comment]
Sorry, just noticed i commited .DS_Store too. Please remove it when you review this PR.

**Comment by AkshayPatwardhan** [comment]
Sorry.... My first PR...

File: config/.DS_Store
Comment (Nightwalker-87) [inline config/.DS_Store:1]: Please delete this file on your current branch and subsequently push it to update this PR.

File: .DS_Store
Comment (Nightwalker-87) [inline .DS_Store:1]: Please delete this file on your current branch and subsequently push it to update this PR.

**Review by Nightwalker-87 — CHANGES_REQUESTED** [review]
Please delete the .DS_Store files on your current branch and subsequently push it to update this PR.

**Comment by AkshayPatwardhan** [comment]
Removed .DS_Store files

**Review by Nightwalker-87 — APPROVED** [review]

**Pull request merged**